### DISCUSSION TITLE
Plotnine Chronicles: Tetris Tactics and Artistic Heatmaps - Where Pixels Play!

### DISCUSSION THREAD

**Discussion by PablaOO7** [has2k1/plotnine#852] [2024 Plotnine Contest]
### Authors

Jaspreet Singh Pabla

### Full description

README link: https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/README.md

### Plots: All plots are available in the repo in the png format.

1. [Heatmap Visualization](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/p9_abstract_heatmap.png)
2. [Tetris Visualization](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/p9_tetris.png)

### Code repository

Code Link: https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/p9_contest_art_viz_notebook.ipynb

**Comment by has2k1** [comment]
## Entry 1

![](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/p9_abstract_heatmap.png?raw=true)

**Comment by has2k1** [comment]
## Entry 2

![](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/First_Entry_Plotnine/p9_tetris.png?raw=true)

**Comment by has2k1** [comment]
@PablaOO7, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
